index.html

```diff
@@ -10,228 +10,307 @@
-            </li>
-            <li>
-                <a href="#"><img src="./images/social/Linkedin.svg" alt="Linkedln" /></a>
-            </li>
-            <li>
-                <a href="#"><img src="./images/social/Vector-M.svg" alt="Gmail" /></a>
-            </li>
-            <li>
-                <a href="#"><img src="./images/social/Github.svg" alt="Github" /></a>
-            </li>
-            <li>
-                <a href="#"> <img src="./images/social/Hand2.svg" alt="Social media"/></a>
-            </li>
-        </ul>    
-    </section>
-                                        <!-- page 2 -->
-                                    <!-- Section 2 : Works -->
-    <section class="work">
-        <ul>
-        <!-- First Card -->
-            <li class="card-container1">
-                <img src="./images/Snapshoot1.png" class="card-container-img1" alt="Snapshoot1">
-
-                <h2 class="title-project1">Tonic</h2>  
-
-                <div class="card-content1">
-                    <ul>
-                        <li class="type1">
-                            <p class="t1">CANOPY</p>
-                            <p class="t2"><img src="./images/Circle.png"/> Back End Dev</p>
-                            <p class="t3"> <img src="./images/Circle.png"/> 2015</p>
-                        </li>  
-                    </ul>                    
-                </div> 
-                    
-                    <p class="text1">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-                    
-                    <ul class="tech1">
-                        <li class="language1">html</li>
-                        <li class="language2"> css</li>
-                        <li class="language3">javascript</li>
+
+    <main class="wrapper">
+        <section class="intro">
+
+            <div class="container-global">
+                <div class="title description">
+                    <h1 class="title">I’m Aima<br>
+                        Glad to see<br>you!</h1>
+                    <p class="description">I’m a software developer! I can help you build a product , feature or website
+                        Look through some of my work and experience! If you like what you see and have a project you
+                        need coded, don’t hestiate to contact me.</p>
+                </div>
+
+                <div class="social container">
+                    <a href="#" class="button connect">LET’S CONNECT</a>
+                    <ul class="social-icons">
+                        <li><img src="/images/social/Twitter.svg" alt=""></li>
+                        <li><img src="/images/social/Linkedin.svg" alt=""></li>
+                        <li><img src="/images/social/Vector-M.svg" alt=""></li>
+                        <li><img src="/images/social/Github.svg" alt=""></li>
+                        <li><img src="/images/social/Hand2.svg" alt=""></li>
```

Comment:
So this is the current comparison of the desings:

| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/221280395-949642d7-a912-4aaa-84b2-dc8421fc4e36.png)| ![image](https://user-images.githubusercontent.com/61361037/221280175-c5482cce-2c64-4d0a-8307-129ff04d21e2.png)|

But when I added `width:100%`, It was working perfectly on all screens. So kindly use this.
![image](https://user-images.githubusercontent.com/61361037/221281499-0cce500f-40d9-47ea-a0e6-0c1ce57b128c.png)